src/webgl/GeometryBuilder.js

```diff
@@ -46,6 +46,8 @@ class GeometryBuilder {
    * transformations.
    */
   addGeometry(input) {
+    this.renderer.uMVMatrix =
```

Comment:
Maybe rather than creating `uMVMatrix` here, we can remove this line and update the lines below to use `uModelMatrix` instead of `uMVMatrix`. In this case we can just get the inverse transpose of the model matrix, by ignoring the camera matrix it's like we're assuming it's an identity matrix and doesn't change anything.